### PR TITLE
Cancel button in wiki editing

### LINKED ISSUE #1983: Cancel button required for ease
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem
Addition of cancel button in edit mode of wiki page so that user need not visit header for going back and he can cancel editing easily.

What actually does happen (screenshot if helpful!): 

## Solution
Addition of hyperlink a tag named as "Cancel" here
can1
 
Where to find the relevant lines of code: https://github.com/publiclab/plots2/blob/master/app/views/wiki/edit.html.erb#L22

You just need to figure out href link,try finding that out yourself first.You can always ask for help.

## Steps to Fix

- [x] claim this issue with a comment here, below, and ask any clarifying questions you need
- [x] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) or go to [the chatroom](http://publiclab.org/chat) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #1983 .

### CODE REVIEW

**Comment by cesswairimu** [comment]
@Gauravano i can't find the correct url to the previous page in the routes. I could use some help thanks

**Comment by Gauravano** [comment]
Hmm..nice try How about trying this `href='/wiki/<%= params["id"] %>'` I think it will solve your problem.

File: app/views/wiki/edit.html.erb [hunk 1]
```diff
@@ -75,6 +75,7 @@
  
       <a id="publish" tabindex="5" class="publish btn btn-primary btn-lg"><%= t('wiki.edit.publish') %></a>
       <a tabindex="6" data-previewing-text="<%= t('wiki.edit.previewing_text') %>" class="btn btn-default btn-lg preview-btn" onClick="$E.toggle_preview()"><%= t('wiki.edit.preview') %></a>
+      <a class="btn btn-danger btn-lg" href="/wiki?id=#{id}">Cancel</a>
```
Comment (Gauravano) [inline app/views/wiki/edit.html.erb]: Awesome.Correct class for cancel button, just replacing href will get it to work.Thanks.

File: app/views/wiki/edit.html.erb [hunk 2]
```diff
@@ -75,6 +75,7 @@
  
       <a id="publish" tabindex="5" class="publish btn btn-primary btn-lg"><%= t('wiki.edit.publish') %></a>
       <a tabindex="6" data-previewing-text="<%= t('wiki.edit.previewing_text') %>" class="btn btn-default btn-lg preview-btn" onClick="$E.toggle_preview()"><%= t('wiki.edit.preview') %></a>
+      <a class="btn btn-danger btn-lg" href="/wiki/<%= params["id"] %>">Cancel</a>
```
Comment (jywarren) [inline app/views/wiki/edit.html.erb]: Sorry for one more request -- could we make this a `btn-default`? Thank you!

Comment (cesswairimu) [reply app/views/wiki/edit.html.erb]: sure

**Comment by jywarren** [comment]
Perfect, thanks so much! :-) 🎈

**Pull request merged**